AUTHORS

```diff
@@ -0,0 +1,17 @@
+# This is the official list of mapp authors for copyright purposes.
+# This file would be distinct from a CONTRIBUTORS file. All contributors
+# own the copyright for their own code in our case, so we don't need a separate one.
+
+# Names should be added to this file as one of
+#     Organization's name
+#     Individual's name <git email address>
+
+# Please keep this list alphabetically sorted.
+
+Angus Pearson <a7cp3ar50n@gmail.com>
+Findlay Smith <findlayksmith@gmail.com>
+Fox Foster <fox@tardis.ed.ac.uk>
+Harjot Singh <harjyot7@gmail.com>
+Harry Reeder <harry@harryreeder.co.uk>
+Qais Patankar <qaisjp@gmail.com>
+William Mathewson <neanias@users.noreply.github.com>
```

Comment:
What's up with this email address?

Reply:
I took it from [here](https://github.com/compsoc-edinburgh/mapp-worker/commit/059d269cd7a66857679f9bba311db81603483afb.patch).  Feel free to update the PR (or lmk your email).

Reply:
I don't know if that would get through to me, but otherwise, my email is wncmathewson@me.com